tests/test_benchmark.py

```diff
@@ -33,19 +34,26 @@ def recenter(conf, box):
 
     return np.array(new_coords)
 
-
 def benchmark(
     label,
     masses,
-    lamb,
+    lamb, # TODO: note lamb unused
```

Comment:
oops - I think for RBFE the intention was for this to be set to 0.5

Reply:
Resolved in https://github.com/proteneer/timemachine/pull/348/commits/f836d025c18667b2caeee21c13dcb6ec12454193